### DISCUSSION TITLE
how to create a plugin on tiddlywiki?

### DISCUSSION THREAD

**Discussion by ghost** [TiddlyWiki/TiddlyWiki5#7442] [Q&A] [answered]
Hi,

I would like to know how to create a plugin on tiddlywiki. how to create a plugin on tiddlywiki?

why?
- I would like to create a [disgus](https://github.com/carlitoplatanito/disgus) plugin on tiddlywiki. 
- I'm trying to create a proof of concept of the nostr decentralized protocol on tiddlywiki, in part because I'm going to be using nostr to comment on my tiddlers

questions?
- Is there any document on how to create a plugin on tiddlywiki?

**Comment by Jermolene** [comment, marked as answer]
Hi @raphaellouisandress there are some developer docs at:

https://tiddlywiki.com/dev/

Perhaps the existing Disqus plugin might make a good starting point:

https://github.com/bimlas/tw5-disqus

It may also make sense as a first step to implement the desired functionality in a static, standalone HTML file, and then to look at how to get it integrated with TiddlyWiki.

**Reply by ghost** [reply]
Hi @Jermolene, thanks for all information here!
